### ISSUE TITLE
Deprecate Mudd-DBS Application

### ISSUE THREAD

**Issue by kevinreiss** [pulibrary/princeton_ansible#5284]
### What maintenance needs to be done?
The [mudd-dbs application](https://github.com/pulibrary/mudd-dbs) has been fully migrated to static-tables and can be sunset. 

### Level of urgency
- [ ] High
- [x] Moderate
- [ ] Low

### Why is this maintenance needed?
- The application is no longer needed. 

### Acceptance criteria
  - [x] Remove staging environment machines
 - [x] Remove production environment machines 
 - [x] Remove Staging Host from DNS
 - [x] Remove Production Host from DNS

Blocked by https://github.com/pulibrary/mudd-dbs/issues/131

**Comment by VickieKarasic** [comment]
lib-mudd-staging1, lib-mudd-prod1, and lib-mudd-prod2 have been powered down. We will wait the recommended 30 days and decommission these machines on 11/4/24.

**Comment by dphillips-39** [comment]
@kevinreiss These VM's have been decommissioned and it looks like there are a couple of deploy configs in the Mudd-DB's repo that need to be deleted https://github.com/search?q=org%3Apulibrary%20lib-mudd&type=code
